templates/theme/styles/base.less

```diff
@@ -277,6 +279,7 @@ atom-text-editor .search-results .marker.current-result .region,
     font-style: italic;
   }
 
+  &.heading,
   &.heading .punctuation.definition.heading {
```

Comment:
There is https://manual.macromates.com/en/language_grammars#naming_conventions, but doesn't go deeper than `.markup.heading`.